android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/QueryTest.java

```diff
@@ -120,15 +120,31 @@ public void check(int n, Result result) throws Exception {
     }
 
     @Test
-    public void testQueryDocumentExpiration() throws CouchbaseLiteException, LiteCoreException, LiteCoreException {
+    public void testQueryDocumentExpiration() throws CouchbaseLiteException, Exception {
+        Date dto5 = new Date(System.currentTimeMillis() + 500L);
+        Date dto10 = new Date(System.currentTimeMillis() + 2000L);
         Date dto20 = new Date(System.currentTimeMillis() + 2000L);
         Date dto30 = new Date(System.currentTimeMillis() + 3000L);
         Date dto40 = new Date(System.currentTimeMillis() + 4000L);
         long  dto60InMS = System.currentTimeMillis() + 6000L;
 
+        MutableDocument doc1 = new MutableDocument("doc");
+        MutableDocument doc10 = new MutableDocument("doc10");
         MutableDocument doc1a = new MutableDocument("doc1");
         MutableDocument doc1b = new MutableDocument("doc2");
         MutableDocument doc1c = new MutableDocument("doc3");
+
```

Comment:
The variable names here are confusing. Can you do just doc1, doc2, doc3, doc4,...?